pkgs/tools/misc/corebootutils/default.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv, fetchurl, zlib, pciutils, coreutils, pmtools, iasl }:
+
+stdenv.mkDerivation rec {
+  pname = "corebootutils";
```

Comment:
Most distros are calling this "coreboot-utils":

https://repology.org/project/coreboot-utils/versions

Reply:
changed, thanks